Client/Application/AppDelegate.swift

```diff
@@ -23,6 +24,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     weak var profile: BrowserProfile?
     var tabManager: TabManager!
     var adjustIntegration: AdjustIntegration?
+    var foregroundTime = 0
```

Comment:
nit: `foregroundTime` makes me think "time in the foreground", i.e. elapsed. I'd call it `timeWhenForegrounded` or something.